lib/chef/data_collector/run_end_message.rb

```diff
@@ -57,6 +57,7 @@ def construct_message(data_collector, status)
             "resources" => all_action_records(action_collection),
             "run_id" => run_status&.run_id,
             "run_list" => node&.run_list&.for_json || [],
+            "expanded_cookbooks" => node&.fetch("cookbooks", {}) || {},
```

Comment:
I think I'd prefer this to be `"cookbooks" => ( node && node["cookbooks"] ) || {}`

1. the same concepts should really have the same names and be kept simple.

2. Hash#fetch is an API that we don't normally use with cookbooks and it creates more cognitive load on the reader (and the reviewer needs to now worry about how well the attributes implementation handles fetch -- while `node["cookbooks"]` is obviously correct)

Reply:
Hey @lamont-granquist 
 
1. As the “cookbooks“ attribute handle and gather on automate config management service so decided to change the attribute name to “expanded_cookbooks“ so that we can avoid mess on Automate side reference https://github.com/chef/automate/blob/4fc6523581f7496174131cdea3bc7ba56747d18e/api/external/cfgmgmt/response/nodes.proto#L77

I was my assumption but okay and did prefer to the same name. 

2. Okay, Gotcha thanks.